app/models/view/Alert.java

```diff
@@ -113,7 +113,15 @@ public void setExtensions(final Map<String, Object> extensions) {
         _extensions = extensions;
     }
 
+    /**
+     * Returns the extensions.
+     *
+     * @return null if extensions are null, an unmodifiable map otherwise.
```

Comment:
I'd prefer to see immutable.  Can we have the setter take and ImmutableMap?  Then you wont have to protect it on the way out (the getter).